src/v3/src/transformer/uischema/setLtrFields.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  FieldElement,
+  TransformStepFn,
+} from '../../types';
+import { traverseLayout } from '../util';
+
+export const setLtrFields: TransformStepFn = (formbag) => {
```

Comment:
Curious - is there a benefit of using this transformer instead of explicitly setting `field.dir` like in `transformRSAAuthenticator`? It seems like we already have one exception that requires us to set it in the transformer.

Reply:
I think the benefit of this is that we can easily add and remove common fields that we want to be ltr only without having to go to each individual transformer and add/remove `field.dir` setting.  (eg. if later we start accepting RTL identifiers, we can simply remove the 'identifier' name from this transformer).  Also when making new transformers, it can be easy to miss adding the `field.dir` for common fields like passcode entry.  But yeah there will be some exceptions for specific non-common use cases where we would need to set the field direction directly in the transformer.  I do think its benefical to have both options

Reply:
Ok, that's a good point. My only concern (not blocking) is that these transforms for uischema run later than the layout transforms (where custom transforms occur) so it's not possible to override the direction behavior for a particular view if the field name matches any of these below. That's probably fine and even desired, but just noting that.